pkg/azurefile/controllerserver.go

```diff
@@ -133,7 +133,20 @@ func (d *Driver) CreateVolume(ctx context.Context, req *csi.CreateVolumeRequest)
 	}
 
 	enableHTTPSTrafficOnly := true
-	retAccount, retAccountKey, err := d.cloud.EnsureStorageAccount(account, sku, accountKind, resourceGroup, location, fileShareAccountNamePrefix, enableHTTPSTrafficOnly)
+
+	var retAccount, retAccountKey string
+	err := wait.ExponentialBackoff(d.cloud.RequestBackoff(), func() (bool, error) {
+		var retErr error
+		retAccount, retAccountKey, retErr = d.cloud.EnsureStorageAccount(account, sku, accountKind, resourceGroup, location, fileShareAccountNamePrefix, enableHTTPSTrafficOnly)
+		isRetriable, sleepTime := isRetriableError(retErr)
+		if isRetriable {
+			klog.Warningf("EnsureStorageAccount(%s) failed with error(%v), sleep %ds to retry", account, retErr, sleepTime)
+			time.Sleep(time.Duration(sleepTime) * time.Second)
```

Comment:
there is already sleep in `ExponentialBackoff`, is it necessary here?

Reply:
I am not sure. Wait for the tests again.

Reply:
According to the result, if we just sleep in ExponentialBackoff, sometimes the sanity tests still fail. 
Following is the bacakoff we set in sanity tests.
```
defaultCloudProviderBackoff         = true
defaultCloudProviderBackoffRetries  = 6
defaultCloudProviderBackoffDuration = 5
```

Reply:
try increase `defaultCloudProviderBackoffDuration` in the sanity test?